cmd/dashboard-redirector/main.go

```diff
@@ -0,0 +1,119 @@
+//
+// Copyright (c) 2018 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	crd "github.com/openshift/ansible-service-broker/pkg/dao/crd"
+
+	"github.com/sirupsen/logrus"
+)
+
+var options struct {
+	BrokerNamespace string
+	Port            int
+}
+
+func init() {
+	flag.IntVar(&options.Port, "port", 1337, "port that the dashboard-redirector should listen on")
+	flag.StringVar(&options.BrokerNamespace, "namespace", "ansible-service-broker", "namespace that the broker resides in")
+	flag.Parse()
+}
+
+var crdDao *crd.Dao
+
+func main() {
+	var err error
+
+	crdDao, err = crd.NewDao(options.BrokerNamespace)
+	if err != nil {
+		panic(fmt.Sprintf("Unable to create crd client - %v", err))
+	}
+
+	logrus.Info("Trying to load batch specs as a sanity check...")
+	specs, err := crdDao.BatchGetSpecs("")
+	if err != nil {
+		panic(fmt.Sprintf("Sanity check failed! -> %v", err))
+	} else {
+		logrus.Info("Sanity check passed! Loaded specs: %v", specs)
```

Comment:
What does this print out?

Reply:
@jmrodri don't the ticks include the newlines?

Reply:
@eriknelson yeah the tick marks will include the new lines.